dvclive/metrics.py

```diff
@@ -49,23 +49,44 @@ def __init__(
                 ) from exception
 
     @staticmethod
-    def from_env():
+    def from_env(existing_logger: "MetricLogger" = None):
         from . import env
 
         if env.DVCLIVE_PATH in os.environ:
             directory = os.environ[env.DVCLIVE_PATH]
```

Comment:
So we're talking about actual environment variables? Specifically `DVCLIVE_PATH`. This comes a bit out of the left field since there's no info on this in https://dvc.org/doc/dvclive/dvclive-with-dvc

Reply:
p.p.s. https://github.com/iterative/dvclive/issues/79 😬

Reply:
Answered in https://github.com/iterative/dvclive/issues/79#issuecomment-844818063, thanks.